simulation/__init__.py

```diff
@@ -21,7 +21,7 @@
 
 from simulation.main import Simulation
 
-__version__ = "0.4.0-alpha"
+__version__ = "0.5.5-alpha"
```

Comment:
If the version number isn't changed, does the package get overwritten? I think a warning might be suitable here if a package of that version already exists.